tgstation.dme

```diff
@@ -2821,6 +2821,7 @@
 #include "code\modules\research\xenobiology\crossbreeding\reproductive.dm"
 #include "code\modules\research\xenobiology\crossbreeding\selfsustaining.dm"
 #include "code\modules\research\xenobiology\crossbreeding\stabilized.dm"
+#include "code\modules\research\xenobiology\crossbreeding\warping.dm"

```

Comment:
well, that's the end of my quick review

let's see how many comments I ended up ma- oh god